contracts/types/Constants.sol

```diff
@@ -44,8 +44,8 @@ uint256 constant MODULE_TYPE_PREVALIDATION_HOOK_ERC1271 = 8;
 uint256 constant MODULE_TYPE_PREVALIDATION_HOOK_ERC4337 = 9;
 
 
-// keccak256("ModuleEnableMode(address module,uint256 moduleType,bytes32 userOpHash,bytes32 initDataHash)")
-bytes32 constant MODULE_ENABLE_MODE_TYPE_HASH = 0xbe844ccefa05559a48680cb7fe805b2ec58df122784191aed18f9f315c763e1b;
+// keccak256("ModuleEnableMode(address module,uint256 moduleType,bytes32 userOpHash,bytes initData)")
+bytes32 constant MODULE_ENABLE_MODE_TYPE_HASH = 0xf6c866c1cd985ce61f030431e576c0e82887de0643dfa8a2e6efc3463e638ed0;
```

Comment:
we changed the typehash to use bytes initData instead of bytes32 initDataHash, we should pass the raw initData bytes directly in the abi.encode call, not its keccak256 hash ?

https://github.com/bcnmy/nexus/blob/release/v1.2.0/contracts/base/ModuleManager.sol#L535